torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -59,13 +59,13 @@ def _put_into_sample(self, sample: Any, id: int, item: Any) -> Any:
 
     def _apply_image_transform(
         self,
-        image: Union[torch.Tensor, PIL.Image.Image, features.Image],
+        image: ImageType,
         transform_id: str,
         magnitude: float,
         interpolation: InterpolationMode,
-        fill: Union[Dict[Type, FillType], Dict[Type, None]],
-    ) -> Any:
-        fill_ = fill[type(image)]
+        fill: Optional[Dict[Type, Optional[FillType]]],
```

Comment:
Shouldn't be 
```suggestion
        fill: Optional[Dict[Type, FillType]],
```
as `FillType` already has None ?

Reply:
Good spot. Accepted.

Reply:
> If FillType can be None, can't we avoid having fill=None and Optional[Dict] ?

No we can't because after the setup is called, the method returns directly `None` instead of just returning a dictionary that gets `None` everytime. Doing the latter is just an extra unnecessary step but if you prefer it I can make the change. See my comment below though about the misalignment of the internal/external signatures.